src/app/modules/core/core.component.html

```diff
@@ -4,7 +4,7 @@
             <img src="assets/logo.png">
         </div>
         <hr>
-        <div class="hovestyle" (click)="goto('dashboard','Dashboard');sidenav.close()">
+        <div class="hovestyle" *ngIf="role == 'Admin' || role == 'HR'" (click)="goto('dashboard','Dashboard');sidenav.close()">
```

Comment:
can we make this a directive *ngIf="role == 'Admin' || role == 'HR'"

because this i am sure would be used in many places

Reply:
let it be ngIF only